spec/stud/interval_spec.rb

```diff
@@ -24,6 +35,59 @@
       end
     end
 
+    it 'should be able to interrupt an interval within same thread' do
+      counter = 0
+
+      Stud.interval(0.5) do
+        counter += 1
```

Comment:
Probably wherever those cool millions Andreeson gave to Github is being stashed. :'(